reports/pvnet_comp.md

```diff
@@ -0,0 +1,269 @@
+
+<table>
+<tr>
+<td><img src="./imgs/pvnet_all_heatmap_MBE_month.png"/></td>
+<td><img src="./imgs/pvnet_all_heatmap_MBE_week.png"/></td>
+</tr>
+<tr>
+<td>Fig 19: PVNet MBE heatmap for Hour vs Month.</td>
+<td>Fig 20: PVNet MBE heatmap for Hour vs Week.</td>
+</tr>
+</table>
+
+The MBE heatmaps across the year confirm the earlier evaluation of a tendency to underpredict during sunrise and sunset hours.
+
+---
+
+## Conclusion
+
+The report evaluates the performance of OCF’s national PV forecast (PVNet) against Sheffield Solar’s PVLive Intraday forecast, using PVLive Updated as the baseline for comparison, focusing on various error metrics across 2022. Due to limitations in data availability, the comparison initially considers overlapping data points for PVLive Intraday and PVNet before extending to all available data for PVNet and PVLive Updated.
+
+PVNet outperforms PVLive Intraday, with a Mean Absolute Error (MAE) of 129.75 MW and a Mean Bias Error (MBE) of 6.30 MW, compared to PVLive Intraday’s MAE of 197.37 MW and MBE of -187.91 MW. This demonstrates statistically significant improvements and increased reliability from PVNet. Additionally, PVNet has greater consistency evident from its lower standard deviations in all metrics compared to PVLive Intraday, indicating that PVNet's errors are more tightly concentrated around the mean.
+
+PVLive Intraday predominantly underpredicts, with a pronounced negative skew in error distribution and the bias peaking in the early summer months. In contrast, PVNet exhibits a symmetrical error distribution but with a tendency to underpredict around sunrise and sunset.
+
+---
+
+## Appendix
+
+### Error Metrics
+
+To compute the error metrics, we utilised the following mathematical formulas:
+
+1. **MAE (Mean Absolute Error)**:
+$$\text{MAE} = \frac{1}{n} \sum_{i=1}^{n} \left| y_{\text{true},i} - y_{\text{pred},i} \right|$$
+
+2. **RMSE (Root Mean Squared Error)**:
+$$\text{RMSE} = \sqrt{\frac{1}{n} \sum_{i=1}^{n} \left( y_{\text{true},i} - y_{\text{pred},i} \right)^2}$$
+
+3. **R^2 (Coefficient of Determination)**:
+$$R^2 = 1 - \frac{\sum_{i=1}^{n} \left( y_{\text{true},i} - y_{\text{pred},i} \right)^2}{\sum_{i=1}^{n} \left( y_{\text{true},i} - \bar{y_{\text{true}}} \right)^2}$$
+
+Where $\bar{y_{\text{true}}}$ is the mean of the true values.
+
+4. **MBE (Mean Bias Error)**:
+$$\text{MBE} = \frac{1}{n} \sum_{i=1}^{n} \left( y_{\text{pred},i} - y_{\text{true},i} \right)$$
+
+### Variability of Error Metrics
+
+The **Standard Deviations** of these metrics are calculated as follows:
+$$\sigma = \sqrt{\frac{1}{n} \sum_{i=1}^{n} \left( \text{Metric}_i - \bar{\text{Metric}} \right)^2}$$
+Where $\bar{\text{Metric}}$ is the mean of the metric across all data points.
+
+The **Standard Error of The Mean** of these metrics are calcuated as follows:
+$$ SE = \frac{\sigma}{\sqrt{n}}$$
+
+## Additional Graphs
+
+The following graphs show results for the root-mean-square deviation (RMSE).
```

Comment:
In each of these graphs, I'd mention which figure in the main text they are similar to. Just to quickly provide context.